### ISSUE TITLE
Section padding/margin

### ISSUE THREAD

**Issue by ineagu** [Codeinwp/gutenberg-blocks#504]
@HardeepAsrani I think this was discussed before, I couldn't find the issue here. The fact that by default the section is adding padding + columns are adding padding, makes theme unusable for me, since every time I would need to bring those to 0.

I guess this makes sense in a full-width context, while when you're just writing a post, is really annoying.

Maybe have some context-based settings along with some defaults that you can set from the options page?

cc @selul

**Comment by HardeepAsrani** [comment]
Changing defaults will break things so we introduced Global Defaults from where you can set your own defaults for the blocks. That should address this. @ineagu 

<img width="785" alt="Screen Shot 2020-11-05 at 8 44 18 PM" src="https://user-images.githubusercontent.com/2649903/98259100-ae590080-1fa7-11eb-9dfc-e303e28aba13.png">

**Comment by ineagu** [comment]
Really really hard to find that for me.

Not sure if we can have a way to open a popup when user set padding to 0 for e.g and ask for default or if user performed same change twice on 2 blocks of the same type

**Comment by HardeepAsrani** [comment]
@selul What do you think? I don't like this idea in particular but I also agree that it's not a very easy thing to find. And this is also another interesting idea https://github.com/Codeinwp/gutenberg-blocks/issues/478

Also, we can change the default, not that I think about it, but instead of changing the defaults in the block, we can just pre-set "Global Defaults" for new users. Might be a hack but a good hack, I'd say. :D

**Comment by selul** [comment]
Yes, I think the challenge is if we can differentiate between newly added blocks and old ones. If we can keep the old style to the existing blocks and use the new defaults for the new ones, would be just great.

**Comment by HardeepAsrani** [comment]
@selul That's what Global Defaults are. They're set when a new block is added. So for the users who haven't added any global defaults, we can just set on their side. Win-win. 🗡️
